practice-one/src/styles/layouts/header.css

```diff
@@ -0,0 +1,38 @@
+.main-header {
+  display: flex;
+  justify-content: space-between;
+  padding: 80px 80px;
+
+  & .header-logo {
+    width: 180px;
+    height: 52px;
+
+    & a { display: block; }
+  }
+
+  & .navbar {
+    display: flex;
+    align-items: center;
+
+    & .navbar-list {
+      display: flex;
+      list-style-type: none;
+    }
+
+    & .navbar-item {
+      margin-right: 48px;
+
+      & .navbar-link {
+        color: var(--color-text-link-normal)
+      }
+
+    }
+
+    & .help-center {
+      border-radius: 6px;
+      padding: 16px 48px;
+      background-color: var(--background-color-primary);
+      color: var(--color-text-link-special);
```

Comment:
No need to override the color here. Because you used it in the class `special-link`